protobuf/v1/schema.proto

```diff
@@ -10,70 +10,62 @@ import "google/api/annotations.proto";
 message ResolveBooleanRequest {
     string flag_key = 1;
     google.protobuf.Struct context = 2;
-    bool default_value = 3;
 }
 message ResolveBooleanResponse {
     bool value = 1;
     string reason = 2;
-    string variant = 3;
```

Comment:
Why did you remove the variant?

Reply:
Variant was never populated by the IEvaluator interface, JSONEvaluator struct or HTTPService. It was also never referenced in the README for any expected responses.
It looks like an artefact from the original proto generation from the existing OpenAPI spec using gnostic as there is reference to it in providers.yaml but is never populated.